eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -52,7 +49,6 @@ stages:
           - AZURE_TEST.*=.*/
           - JavaTestVersion=(.*1)\.17(.*)/$1.11$2
         BuildParallelization: 1
```

Comment:
AdditionalBuildOptions were being passed so to enable spark shading in testing in the CI. 
Now we are removing them. Is there a different way to shade cosmos spark libraries now?

Reply:
From the other comment shading Spark is always happening now